baseapp-auth/baseapp_auth/graphql/object_types.py

```diff
@@ -48,7 +48,7 @@ def resolve_profiles(self, info):
                 return Profile.objects.none()
             return Profile.objects.filter(
                 Q(owner_id=info.context.user.id) | Q(members__user_id=info.context.user.id)
-            )
+            ).distinct()
```

Comment:
This ensures that profiles are only displayed once when switching profiles (calling `.filter` with a condition that involves multiple databases might produce duplicate entries)